### PR TITLE
matagenome-atlas v.2.1.0

### PR DESCRIPTION
**Pull request by SilasK**
snakemake>=5.4.5

test cli

* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [ ] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [x] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

**Comment by SilasK** [comment]
I build this recipe successfully except for the testing of the command line interface (click), which I integrated in the test. 
But anyway it is good to have a review. 

Thank you.

File: recipes/metagenome-atlas/meta.yaml [hunk 1]
```diff
@@ -1,48 +1,49 @@
...
+  sha256: {{ sha256 }}
 
 build:
-  skip: True  # [osx]
-  number: 1
+  number: 0
   entry_points:
     - atlas=atlas.atlas:cli
-
+  noarch: True
```
Comment (bgruening) [inline recipes/metagenome-atlas/meta.yaml]: this should be `noarch: python` I guess

**Comment by epruesse** [comment]
@biocodnabot lint

**Comment by epruesse** [comment]
@BiocondaBot lint

**Pull request merged**